Description: Fix use C.UTF-8 as hardcoded internal locale
 Debian always provides C.UTF-8, only often en_US.UTF-8, and not en_US.utf-8.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2016-12-21

--- a/louloulibs/utils/time.cpp
+++ b/louloulibs/utils/time.cpp
@@ -24,7 +24,7 @@
   std::tm t = {};
 #ifdef HAS_GET_TIME
   std::istringstream ss(stamp);
-  ss.imbue(std::locale("en_US.utf-8"));
+  ss.imbue(std::locale("C.UTF-8"));
 
   std::string timezone;
   ss >> std::get_time(&t, format) >> timezone;
